### PR TITLE
Fixing bad xref in Virt Post-install

### PR DESCRIPTION
**Pull request by mburke5678**
Noticed a bad link in the Virtualization -> -> Post-install -> Network configuration page. This PR is to fix that link. 

4.14+.  Assembly not in prior versions.

No associated issue

No QE required

### CODE REVIEW

**Comment by mburke5678** [comment]
@apinnick PTAL. 

The [Network configuration link](https://docs.openshift.com/container-platform/4.14/virt/post_installation_configuration/virt-post-install-config.html) is not working.  The XREF looks wrong to me; but, perhaps it is some fancy HTML formatting that I am not familiar with.

File: virt/post_installation_configuration/virt-post-install-config.adoc [hunk 1]
```diff
@@ -12,7 +12,7 @@ The following procedures are typically performed after {VirtProductName} is inst
 
 * xref:../../virt/post_installation_configuration/virt-node-placement-virt-components.adoc#virt-node-placement-virt-components[Node placement rules for {VirtProductName} Operators, workloads, and controllers]
 
-* xref:../../virt/post_installation_configuration/./.virt-network-config.adoc#virt-post-install-network-config[Network configuration]:
+* xref:../../virt/post_installation_configuration/virt-post-install-network-config.adoc#virt-post-install-network-config[Network configuration]:
```
Comment (apinnick) [inline virt/post_installation_configuration/virt-post-install-config.adoc:15]: Not sure I understand your question. I evidently made a mistake with this xref. Your change is correct.

Comment (mburke5678) [reply virt/post_installation_configuration/virt-post-install-config.adoc:15]: @apinnick I wanted to make sure the formatting was not intentional, perhaps some formatting I'm not familiar with.

Comment (apinnick) [reply virt/post_installation_configuration/virt-post-install-config.adoc:15]: @mburke5678  Definitely not intentional! Just my poor, tired, senile brain having hiccups. :-)

**Pull request merged**